_episodes/09-working-with-sql.md

```diff
@@ -66,7 +66,7 @@ con = sqlite3.connect("data/portal_mammals.sqlite")
 cur = con.cursor()
 
 # Return all results of query
-SELECT plot_id FROM plots WHERE plot_type="Control"')
+cur.execute('SELECT plot_id FROM plots WHERE plot_type="Control"')
```

Comment:
It might be better to do it in a separate PR

Reply:
Just rolled it back.